### PR TITLE
Perf/get target kind

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
I use temp `main.rs` to bench
main branch: 
```bash
    Finished release [optimized] target(s) in 2.63s
     Running `target/release/nodejs-resolver`
2.47245ms
```

current branch: 
```bash
Finished release [optimized] target(s) in 7.50s
     Running `target/release/nodejs-resolver`
216.629µs
```

**Comment by bvanjoi** [comment]
Thanks for the optimization, benchmark have been added to the main branch, could you  rebase the main branch and commit again?

File: src/main.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+use std::time::Instant;
```
Comment (bvanjoi) [inline src/main.rs:1]: Maybe this file could be deleted after rebase `main` branch.

**Comment by IWANABETHATGUY** [comment]
It seems I don't have permission to trigger benchmark?

**Comment by bvanjoi** [comment]
> It seems I don't have permission to trigger benchmark?

Rebase and try it again? I had use personal token to replace `GITHUB_TOKEN`.

**Comment by bvanjoi** [comment]
> > It seems I don't have permission to trigger benchmark?
> 
> Rebase and try it again? I had use personal token to replace `GITHUB_TOKEN`.

Sorry, I have some problems writing the code, I will change it again....😅

**Comment by bvanjoi** [comment]
> > > It seems I don't have permission to trigger benchmark?
> > 
> > 
> > Rebase and try it again? I had use personal token to replace `GITHUB_TOKEN`.
> 
> Sorry, I have some problems writing the code, I will change it again....😅

Try it again? @IWANABETHATGUY

**Comment by IWANABETHATGUY** [comment]
> 
Would you mind trigger ci again?

**Comment by IWANABETHATGUY** [comment]
image

**Comment by bvanjoi** [comment]
Helpless, merged it  directly.. 😢

**Comment by IWANABETHATGUY** [comment]
according to the Ci in main branch, 
image

We gain 20% speed up

**Pull request merged**